src/components/MenuTab.tsx

```diff
@@ -0,0 +1,46 @@
+import clsx from "clsx";
+import type { ReactNode } from "react";
+import { useState } from "react"
+
+export type MenuItem = {
+  title: string
+  content: ReactNode
+  onChange?: (index: number) => void
+}
+
+export const MenuTab = ({ items, lightBackground, isSelected }: { items: Array<MenuItem>, lightBackground?: boolean, isSelected?: number }) => {
+  const [ currentIdx, setCurrentIdx ] = useState(0)
+  const equalSplitValue = 100 / items.length + "%"
+  const selectedIdx = isSelected ? isSelected : currentIdx
+  return (
+    <>
+      <div className="flex flex-row justify-center overflow-hidden">
+        { items.map((item, idx) => (
+          <>
+            <div
+              key={ item.title + idx + "--tab" }
+              className={clsx("btn rounded-none", {
```

Comment:
Could we adjust the border radius, so the tabs look a bit more like the buttons?

I was thinking something along with this:
___________________________

(  General     |   Commits  )
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾

(Sorry for my ugly ASCII art)

Also, I don't really like how the underline looks.

Reply:
It is a bad practive to use `div`'s for button purposes, prefer using "<button>". If you absolutely need to use a div for a button, remember to add `role="button"`, such that it is still tabbable.

Reply:
I removed the underline. Is that what you meant with the border radius?
![image](https://github.com/git-truck/git-truck/assets/52498300/a48add8c-ba1d-4e82-bf40-5bb293c17f19)

Reply:
Yeah, that looks great!